rclc/test/rclc/test_action_client.cpp

```diff
@@ -127,6 +127,10 @@ TEST(Test, rclc_action_client) {
   // clean up
   rc = rclc_action_client_fini(&action_client, &node);
   EXPECT_EQ(RCL_RET_OK, rc);
+  if (rcl_logging_rosout_enabled()) {
```

Comment:
we do not need to check the impl object if that is actually created since `rcl_logging_rosout_fini_publisher_for_node` always return `RCL_RET_OK` when it is not initialized. (and so is elsewhere.)